src/dso_api/dynamic_api/fields.py

```diff
@@ -46,7 +49,7 @@ def to_representation(self, value):
             "request" in self.parent.context
             and self.parent.context["request"].versioned
         ):
-            value = value.split("_")[0]
+            value = PK_SPLIT.split(value)[0]
```

Comment:
Misschien nog een test voor de split toevoegen?

Reply:
Goed plan. Al is het tijdelijk, toch zorgen dat het zo stabiel mogelijk blijft.